package.json

```diff
@@ -35,10 +35,8 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@popperjs/core": "^2.2.2",
-    "@types/puppeteer": "^2.0.1",
-    "@types/uuid": "^7.0.3",
```

Comment:
Types never need to be build dependencies, AIUI, so I moved these to devDependencies.